doc/features.md

```diff
@@ -0,0 +1,74 @@
+Network Module Features
+-----------------------
+
+Network Services
+======================
+
+- wicked
+- Network Manager
+- no network service
```

Comment:
we're able to handle even old netconfig .. when it is installed. But i don't know if it is still doable in SLE-12.